### DISCUSSION TITLE
doris.2.0.14版本show frontends耗时很慢 30s，其他命令都是毫秒级

### DISCUSSION THREAD

**Discussion by baimafeima-yf** [apache/doris#40589] [A - General / Q&A] [answered]
![doris问题3](https://github.com/user-attachments/assets/8479fe76-f7da-48f1-8089-478de8ea4e86)
fe日志也没有明显报错，/etc/resolv.conf也是空的，没配其他地址
![doris问题1](https://github.com/user-attachments/assets/d3e0f355-cbb1-488c-8222-658381331e5d)
![doris问题2](https://github.com/user-attachments/assets/449f473e-1a96-4e16-8df3-a5bb723df95b)

**Comment by ixzc** [comment, marked as answer]
done. https://ask.selectdb.com/questions/D1AP1/E1SP1?commentId=10070000000006961
